MQTTClient.podspec

```diff
@@ -1,13 +1,13 @@
 Pod::Spec.new do |mqttc|
 	mqttc.name         = "MQTTClient"
-	mqttc.version      = "0.14.0"
+	mqttc.version      = "0.14.1"
```

Comment:
I'm guessing this should be an increment of the minor version, i.e. bump to 0.15.0 (or maybe that would be another PR for a release) since this is a feature enhancement rather than a bug fix.